libraries/cyclestreets-view/src/main/java/net/cyclestreets/routing/Segment.java

```diff
@@ -53,6 +55,7 @@
           final String turnInstruction,
           final boolean walk,
           final String runningTime,
+          final int time,
```

Comment:
Given the usage of this is in parallel with `cumulativeDistance`, should we not call it `cumulativeTime`?

Reply:
I think I was using the existing name, but I agree, cumulativeTime would be clearer, so I'll look to change the name in the existing code as well.